jwst/regtest/test_nirspec_fs_spec3.py

```diff
@@ -42,3 +45,22 @@ def test_nirspec_fs_spec3(run_pipeline, rtdata_module, fitsdiff_default_kwargs,
     # Compare the results
     diff = FITSDiff(rtdata.output, rtdata.truth, **fitsdiff_default_kwargs)
     assert diff.identical, diff.report()
+
+    if "s2d" in output:
+        # Compare the calculated wavelengths
+        tolerance = 1e-03
+        dmt = datamodels.open(rtdata.truth)
+        dmr = datamodels.open(rtdata.output)
+        if isinstance(dmt, datamodels.MultiSlitModel):
+            for slit_idx, slit in enumerate(dmt.slits):
```

Comment:
The tests look good. One possible improvement is to not assume the files have the same order of slits but use `slit.name` to access them. For example something like:
```
names = [s.name for s in s2d.slits]
for name in names:
    sidx = [(s.wcs, s.wavelength) for s in s2d.slits if s.name==name]
...
```
Or something similar

Reply:
Thank you for this suggestion! It's now included in the code.